modules/maven/src/main/java/org/eclipse/sw360/antenna/maven/HttpRequester.java

```diff
@@ -53,7 +53,7 @@ public HttpRequester(ProxySettings proxySettings) {
         File localJarFile = targetDirectory.resolve(jarBaseName).toFile();
 
         if (localJarFile.exists()) {
-            LOGGER.info("The file " + localJarFile + " already exists and won't be downloaded again");
+            LOGGER.debug("The file " + localJarFile + " already exists and won't be downloaded again");
```

Comment:
I would make this a warning

Reply:
Why should this warn the user? For me it's a warning, if the jar could't not be downloaded or something like that. But if the jar already locally exists, everything is fine.